jenkins-pipelines/manager/ubuntu22-manager-upgrade.jenkinsfile

```diff
@@ -0,0 +1,21 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    manager: true,
```

Comment:
what's this boolean for ? I don't see it ever being used

it's a pipeline called `managerPipeline` why someone need a extra boolean to say that again ?

Reply:
seems like it's create when the new `managerPipeline` but without any use, and copied again and again
to multiple pipelines, @ShlomiBalalis I would recommend opening a PR to remove this unused boolean.

Reply:
Huh, didn't notice it. Yeah it seems like an old remnant that is pretty useless now.
I removed it from the current pr and have taken a note to remove it from the rest of them.